_includes/markdown/Markup.md

```diff
@@ -257,6 +257,21 @@ ARIA attributes can be added with JavaScript to help dynamically add context to
 
 You can see how effortless it is to make our tabbed interface accessible to screen readers. All we need to do is add context.
 
+#### Skip to Content Links
+
+[Skip to content links](https://developer.wordpress.org/themes/functionality/accessibility/#skip-links), also known as "skip links", allow screen readers and assistive technologies to skip directly to content. Skip links are a WordPress Core standard that 10up aims to implement across all projects.
+
+Skip Links are usually placed in the header of a WordPress theme, before the site navigation. While they usually skip to the main content of the site, they can also skip to other navigations as well as the footer. Skip Links make use of the ```screen-reader-text``` class to position the links off the screen, and on ```:focus```, the links become visible for sighted keyboard users. The CSS technique used to show and hide these links, hide them from sighted users, but will still keep them accessible for screen readers. Here's what a skip link looks like:
```

Comment:
1) This paragraph doesn't need to be exclusive to WordPress.

2) Why would a "skip to content link" skip to the footer?

3) This sentence is a bit confusing: "The CSS technique used to show and hide these links, hide them from sighted users, but will still keep them accessible for screen readers."